README.md

```diff
@@ -79,7 +79,8 @@ You can subscribe and join the mailing list on [Google Groups](https://groups.go
 
 OCI discussion happens in the following chat rooms, which are all bridged together:
 
-- #general channel on [OCI Slack](https://chat.opencontainers.org/)
+<!-- https://chat.opencontainers.org is currently down: https://github.com/opencontainers/.github/issues/39 -->
+- #general channel on [OCI Slack](https://join.slack.com/t/opencontainers/shared_invite/zt-vzxfzkkl-CbsYY9zwMhSTkXlvvhmBVA)
```

Comment:
https://communityinviter.com/apps/opencontainers/join-the-oci-community
```suggestion
- #general channel on [OCI Slack](https://communityinviter.com/apps/opencontainers/join-the-oci-community)
```
or maybe https://opencontainers.org/community/overview/#chat

Reply:
Thanks, added https://opencontainers.org/community/overview/#chat